src/software/ai/passing/pass_generator.cpp

```diff
@@ -9,26 +9,30 @@
 using namespace Passing;
 
 PassGenerator::PassGenerator(const World& world, const Point& passer_point,
-                             const PassType& pass_type)
+                             const PassType& pass_type, bool running_deterministically)
     : updated_world(world),
       world(world),
       passer_robot_id(std::nullopt),
       optimizer(optimizer_param_weights),
       passer_point(passer_point),
       best_known_pass({0, 0}, {0, 0}, 0, Timestamp::fromSeconds(0)),
       target_region(std::nullopt),
-      random_num_gen(random_device()),
+      random_num_gen(13),
```

Comment:
Is this is a magic number? Maybe pull it out as a constant and add a comment

Reply:
Added a comment:
```
      // We initialize the random number generator with a specific value to 
      // allow generated passes to be deterministic. The value used here has
      // no special meaning.

```